secant/Dependencies/SDKSynchronizer/SDKSynchronizerInterface.swift

```diff
@@ -18,24 +18,11 @@ extension DependencyValues {
 }
 
 enum SDKSynchronizerState: Equatable {
-    case unknown
-    case transactionsUpdated
-    case started
     case progressUpdated
-    case statusWillUpdate
-    case synced
+    case started
     case stopped
-    case disconnected
-    case syncing
-    case downloading
-    case validating
-    case scanning
-    case enhancing
-    case fetching
-    case minedTransaction
-    case foundTransactions
-    case failed
-    case connectionStateChanged
```

Comment:
how do you know of errors?

Reply:
same here, state of the synchronizer stayed the same, secant never supported all of it, so I cleaned up the internal secant enum